jbpm-designer-backend/src/main/java/org/jbpm/designer/web/server/TaskFormsEditorServlet.java

```diff
@@ -123,94 +126,109 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
 	 }
 
 	 private JSONObject storeTaskFormInRepository(String formType, String taskName, String packageName, String formValue, Repository repository) throws Exception{
+         if(formType.equals(FORMMODELER_FILE_EXTENSION) || formType.equals(FORMMODELER_PREVIEW_FILE_EXTENSION)) {
+             repository.deleteAssetFromPath(packageName + "/" + taskName + TASKFORM_NAME_EXTENSION + "." + formType);
 
-        repository.deleteAssetFromPath(packageName + "/" + taskName + TASKFORM_NAME_EXTENSION + "." + formType);
+             AssetBuilder builder = AssetBuilderFactory.getAssetBuilder(Asset.AssetType.Byte);
+             builder.location(packageName)
+                     .name(taskName + TASKFORM_NAME_EXTENSION)
+                     .type(formType)
+                     .content(formValue.getBytes("UTF-8"));
 
-        AssetBuilder builder = AssetBuilderFactory.getAssetBuilder(Asset.AssetType.Byte);
-        builder.location(packageName)
-                .name(taskName + TASKFORM_NAME_EXTENSION)
-                .type(formType)
-                .content(formValue.getBytes("UTF-8"));
+             repository.createAsset(builder.getAsset());
 
-        repository.createAsset(builder.getAsset());
+             Asset newFormAsset = repository.loadAssetFromPath(packageName + "/" + taskName + TASKFORM_NAME_EXTENSION + "." + formType);
 
-        Asset newFormAsset =  repository.loadAssetFromPath(packageName + "/" + taskName + TASKFORM_NAME_EXTENSION + "." + formType);
-
-        String uniqueId = newFormAsset.getUniqueId();
-        if (Base64.isBase64(uniqueId)) {
-            byte[] decoded = Base64.decodeBase64(uniqueId);
-            try {
-                uniqueId =  new String(decoded, "UTF-8");
-            } catch (UnsupportedEncodingException e) {
-                e.printStackTrace();
-            }
-        }
+             String uniqueId = newFormAsset.getUniqueId();
```

Comment:
This decoding logic is duplicated (I see it 3 times just in this PR). Could you please extract it to separate method and reuse?